coalib/coala_main.py

```diff
@@ -52,6 +54,10 @@ def run_coala(log_printer=None,
     :return:                        A dictionary containing a list of results
                                     for all analyzed sections as key.
     """
+    # Configure logging
+    with open('logging_config.json') as fd:
+        logging.config.dictConfig(json.load(fd))
```

Comment:
the you should add a constant for the filenme in Constants and use the calib root constant for getting the filename